### PR TITLE
Adding HelloWorld sample for Kotlin

### PR DESCRIPTION
**Pull request by giusedroid**
## Description
This PR adds an HelloWorld sample written in Kotlin. I translated the HelloWorld sample from Java to Kotlin.

## Motivation and Context
A partner with expertise in Android development asked if it was possible to provide a Kotlin example.

## Testing
1. I built the skill with the current config using gradle
2. I deployed the sample skill to a Lambda function in one of my AWS accounts and tested manually.

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [x] My code follows the code style of this project
- [ ] My change requires a change to the documentation
- [ ] I have updated the documentation accordingly
- [x] I have read the **README** document
- [ ] I have added tests to cover my changes
- [ ] All new and existing tests passed

## License
- [x] By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

[issues]: https://github.com/alexa/alexa-skills-kit-sdk-for-java/issues
[license]: http://aws.amazon.com/apache2.0/
[cla]: http://en.wikipedia.org/wiki/Contributor_License_Agreement

### CODE REVIEW

**Comment by hannesa2** [comment]
Yes please, I would love to see this on master

File: samples/helloworld-kt/build.gradle.kts [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+
+plugins {
+    kotlin("jvm") version "1.5.10"
```
Comment (hannesa2) [inline samples/helloworld-kt/build.gradle.kts]: Time is moving fast, now it's `1.6.10`

File: samples/helloworld-kt/build.gradle.kts [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+group = "alexa-skills-kit-samples"
+version = "1.0-SNAPSHOT"
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    implementation("com.amazon.alexa:ask-sdk:2.40.0")
```
Comment (hannesa2) [inline samples/helloworld-kt/build.gradle.kts]: `2.42.1`

File: samples/helloworld-kt/build.gradle.kts [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    implementation("org.apache.logging.log4j:log4j-core:2.8.2")
```
Comment (hannesa2) [inline samples/helloworld-kt/build.gradle.kts]: This is important `2.17.1` because of Log4j bug

**Review by hannesa2 — DISMISSED** [review]
Additional a Github action to build it would be fine. I could do it, if it's on master

File: samples/helloworld-kt/build.gradle.kts (hunk 1 above)
Comment (giusedroid) [reply samples/helloworld-kt/build.gradle.kts]: bumped up :)

File: samples/helloworld-kt/build.gradle.kts (hunk 2 above)
Comment (giusedroid) [reply samples/helloworld-kt/build.gradle.kts]: bumped up

File: samples/helloworld-kt/build.gradle.kts (hunk 3 above)
Comment (giusedroid) [reply samples/helloworld-kt/build.gradle.kts]: bumped up. Thanks for checking on this.

**Comment by giusedroid** [comment]
bumped all required deps.

**Review by hannesa2 — APPROVED** [review]

**Comment by doiron** [comment]
We are not supporting Kotlin at the moment.  will close this PR until we decide to add support

**Comment by hannesa2** [comment]
But you should support Kotlin, It is de facto standard for Android since ~2019
@doiron

**Pull request closed without merging**